src/me/StevenLawson/TotalFreedomMod/Commands/Command_smite.java

```diff
@@ -15,13 +18,22 @@
 {
     @Override
     public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
-    {
+    {   
+        final Player player = getPlayer(args[0]);
+        
+        final ArrayList<Player> cooldown = new ArrayList<Player>();
+        
+        if (cooldown.contains(player))
```

Comment:
`player` can be null before this if statement is reached. The null check is currently located at line 38 (26 before).